### DISCUSSION TITLE
2022 Testing Data Availability

### DISCUSSION THREAD

**Discussion by ProjectF1** [theOehrly/Fast-F1#93] [Q&A]
Hi there,
Do we know at this stage whether 2022 testing data will be available via FastF1? Will this be impacted by the decision to have testing as a closed-door event that may not be streamed?
Any insights would be much appreciated.
Thanks!
ProjectF1

**Comment by theOehrly** [comment]
Hi, I cannot give you a definite answer to that. I haven't read any specifics about whether or not live timing will be made available in the official app for this years testing events. And that is the most important point.

For the first test, journalists are allowed but there's no live broadcast. I'd say maybe.

The second test, I'm guessing it will be available. But we will see.

Also, it may be that I need to adapt FastF1 to any changes that were introduced by F1 over the winter. Which I will find out when the first session starts.

**Reply by ProjectF1** [reply]
Understood, thanks for the fast answer. I'll keep my eyes peeled for any further information and will check back here as well.

Thanks again!

**Comment by siddharth-tewari** [comment]
@theOehrly @ProjectF1 Hello, I made a website to see car launch dates.
Link
https://siddharthtewari.pythonanywhere.com/F1-2022-Car-Launch

**Comment by zyxwl2015** [comment]
Hi, now that the testing has started, do we know if testing data this week is available on Fastf1?

**Reply by theOehrly** [reply]
I've found no way to pull the data so far. It's also not available in the official app. Therefore, I'm fairly certain that there is no way to get the data for this first week of testing.
